src/output-json.c

```diff
@@ -707,6 +710,70 @@ void CreateJSONFlowId(json_t *js, const Flow *f)
     }
 }
 
+static inline int JSONFormatAndAdd(json_t *parent, const char *key,
+                                   uint8_t *val, bool is_array) {
```

Comment:
Function should probably have something in the name about mac/eth addresses.  Can also just return void.

Reply:
Done.